README.md

```diff
@@ -6,6 +6,12 @@
 
 <img src="https://user-images.githubusercontent.com/847532/169708857-5aed1ebb-76c4-43de-8309-469c0e8cf2f2.jpg" alt="Hasura Change Summary example comment" width="689">
 
+## Features
+
+This action currently supports changes to database table metadata including row-level and column-level permissions.
+
+[Other metadata](https://hasura.io/docs/latest/graphql/core/migrations/reference/metadata-format/) such as actions, cron triggers, and remote schemas are not currently supported.
```

Comment:
4874fb3efb404583ea9726b613feb3a37fe88c7c makes some minor wording updates; let me know if that reads alright for you.